spec/fixtures/apply/device_test/lib/puppet/type/fake_device.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'puppet/resource_api'
+
+Puppet::ResourceApi.register_type(
+  name: 'fake_device',
+  docs: 'fake device for testing puppet-device',
+  attributes: {
+    ensure: {
+      type: 'Enum[present, absent]',
+      desc: 'Whether this apt key should be present or absent on the target system.',
```

Comment:
apt key?

Reply:
This was copied from panos. I can update the desc.